ui/src/components/Common/Demo/InputTextDemo.vue

```diff
@@ -0,0 +1,20 @@
+<template>
```

Comment:
This demo page will eventually become the Skeleton page and then eventually will house the entire experience.

So we'll have to change the name of it eventually, but for now this is acceptable. :)

Reply:
Okay